include/SFML/Graphics/Texture.hpp

```diff
@@ -569,7 +573,16 @@ class SFML_GRAPHICS_API Texture : GlResource
     /// \param coordinateType Type of texture coordinates to use
     ///
     ////////////////////////////////////////////////////////////
-    static void bind(const Texture* texture, CoordinateType coordinateType = Normalized);
+    static void bind(const Texture* texture, sf::CoordinateType coordinateType = sf::CoordinateType::Normalized);
```

Comment:
`sf::CoordinateType::Normalized` is C++11, we should use `sf::Normalized`.

Reply:
My C++03 is so rusty! Fixed.

Reply:
🦀